app/views/all_casa_admins/sessions/new.html.erb

```diff
@@ -31,7 +31,7 @@
 
           <%= render "all_casa_admins/shared/links" %>
 
-          <br/>
+          <br>
```

Comment:
I'm curious, why is it better to omit the slash here?

Reply:
This is a default of the linting library. I recommend reading the linting library docs for information about the choices it makes. https://github.com/Shopify/erb-lint#SelfClosingTag

Reply:
In HTML5, it's not necessary to use self-closings tags for void elements. `erb-lint` [made this change based on HTML5 style](https://github.com/Shopify/erb-lint#SelfClosingTag)